lib/AST/ASTImporter.cpp

```diff
@@ -971,6 +971,23 @@ bool ASTNodeImporter::hasSameVisibilityContext(T *Found, T *From) {
   return false;
 }
 
+template <>
+bool ASTNodeImporter::hasSameVisibilityContext(VarTemplateDecl *Found,
```

Comment:
Would it work if we called the primary template with the params Found->getTemplatedDecl() and From->getTemplatedDecl() ?
If yes, then we'd not need this specialization.